cilium-dbg/cmd/encrypt_status_test.go

```diff
@@ -14,8 +14,9 @@ var _ = Suite(&EncryptStatusSuite{})
 const procTestFixtures = "fixtures/proc"
 
 func (s *EncryptStatusSuite) TestGetXfrmStats(c *C) {
-	errCount, m := getXfrmStats(procTestFixtures)
-	currentCount := 0
+	errCount, m, err := getXfrmStats(procTestFixtures)
+	c.Assert(err, Equals, nil)
+	currentCount := int64(0)
```

Comment:
```suggestion
	currentCount := 0
```
Maybe there's a good reason you set this to `int64`, but it's causing unit test errors here.

Reply:
I used `int64` because of OpenAPI generates `int64` for the `integer` type.
Fixed.

Also, I've converted the global `countErrors` variable in the `encrypt_status.go` file into local.
All the changes were pushed in my second commit with updated comment.

Now all tests green.
Thank you, @nathanjsweet!